lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -39,7 +39,8 @@ def self.determine_next_version_using_labels(repo_name, github_token, rate_limit
       end
 
       def self.auto_generate_changelog(repo_name, github_token, rate_limit_sleep)
-        old_version = Actions.sh("git describe --tags --abbrev=0").strip
+        Actions.sh("git fetch --tags")
```

Comment:
Added this since a couple of times now, I've tried to bump but forgot to fetch latest tags first.

Reply:
Note that I didn't add this in the `determine_next_version_using_labels` since that method is currently only intended to be used in automated methods so it seemed unnecessary